package.json

```
@@ -8,7 +8,7 @@
   "dependencies": {
     "npm-registry-fetch": "^11.0.0"
   },
-  "main": "index.js",
+  "main": "lib",
```

this is a tiny nit but it is likely better to be explicit here. The node.js resolution algorithm will end up doing something like 5-7 different file stats to resolve the index if we don't explicitly state the path
```suggestion
  "main": "./lib/index.js",
```